pkg/lwip/contrib/netdev/lwip_netdev.c

```diff
@@ -52,8 +53,8 @@
 #define WPAN_IFNAME2 'P'
 
 static kernel_pid_t _pid = KERNEL_PID_UNDEF;
-static char _stack[LWIP_NETDEV_STACKSIZE];
-static msg_t _queue[LWIP_NETDEV_QUEUE_LEN];
+static WORD_ALIGNED char _stack[LWIP_NETDEV_STACKSIZE];
```

Comment:
This is actually an unrelated sneak in for which I'm too lazy to open a separate PR. `thread_create()` will align the stack to word boundaries for all architectures by dropping one word of stack space. Having the stack aligned to word boundaries by the linker results in one word more stack space being available.